api/src/pcapi/scripts/beneficiary/import_test_users.py

```diff
@@ -325,30 +323,19 @@ def create_users_from_google_sheet(document_id: str, update_if_exists: bool = Fa
 
 
 @blueprint.cli.command("import_test_users")
-@optgroup.group("User data sources", cls=RequiredMutuallyExclusiveOptionGroup)
-@optgroup.option(
-    "-d",
-    "--default",
-    is_flag=True,
-    help="Import from default Google document set in IMPORT_USERS_GOOGLE_DOCUMENT_ID",
-)
-@optgroup.option("-f", "--filename", help="Path to the CSV file to import")
-@optgroup.option("-g", "--google", "google_id", help="Google document id")  # type: ignore [arg-type]
+@click.option("-f", "--filename", type=str, required=True, help="Path to the CSV file to import")
```

Comment:
Il ne faut pas que ce paramètre soit `required` !
```suggestion
@click.option("-f", "--filename", type=str, required=False, help="Optional path to the CSV file to import (default is from Google Drive)")
```